lib/ansible/modules/notification/mail.py

```diff
@@ -241,9 +255,27 @@ def main():
     secure_state = False
     sender_phrase, sender_addr = parseaddr(sender)
 
+    gnupg_recipients = module.params.get('gnupg_recipients')
+    gnupg = gnupg_recipients or module.params.get('gnupg')
+
     if not body:
         body = subject
 
+    if gnupg:
+        import gnupg
```

Comment:
It should verify that gnupg is installed and can be imported. See for example https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/packaging/os/apt_repository.py#L124 on how we do it

Reply:
Can you confirm the fix is OK, or whether I need the import in the top-level?